### PR TITLE
Fix up FQ and race condition issues with volatile tasks work [VS-478]

### PR DESCRIPTION
**Pull request by mcovarr**
Integration test successful https://app.terra.bio/#workspaces/broad-firecloud-dsde/VS-415%20GVS%20Quickstart%20Default%20Extract%20Scatter/job_history/73ac71db-0488-46be-a8e8-7f00e795edec

### CODE REVIEW

File: scripts/variantstore/wdl/GvsCreateAltAllele.wdl [hunk 1]
```diff
@@ -11,6 +11,8 @@ workflow GvsCreateAltAllele {
     String? service_account_json_path
   }
 
+  String fq_alt_allele_table = "~{dataset_name}.alt_allele"
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsCreateAltAllele.wdl:14]: do we want project name in this too to keep it consistent?

Comment (mcovarr) [reply scripts/variantstore/wdl/GvsCreateAltAllele.wdl]: yup, that turns out to be a required fix.

File: scripts/variantstore/wdl/GvsCreateAltAllele.wdl [hunk 2]
```diff
@@ -11,6 +11,8 @@ workflow GvsCreateAltAllele {
     String? service_account_json_path
   }
 
+  String fq_alt_allele_table = "~{project_id}.~{dataset_name}.alt_allele"
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsCreateAltAllele.wdl:14]: bug 1: alt allele table must be fully qualified

File: scripts/variantstore/wdl/GvsExtractCallset.wdl [hunk 3]
```diff
@@ -110,7 +110,7 @@ workflow GvsExtractCallset {
   call Utils.GetBQTableLastModifiedDatetime as FilterSetInfoTimestamp {
        input:
        query_project = project_id,
-       fq_table = "filter_set_info",
+       fq_table = "~{fq_gvs_dataset}.filter_set_info",
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsExtractCallset.wdl:113]: bug 3: same as bug 1, BQ table name must be fully qualified here.

File: scripts/variantstore/wdl/GvsCreateAltAllele.wdl [hunk 4]
```diff
@@ -27,8 +29,9 @@ workflow GvsCreateAltAllele {
 
   call Utils.GetBQTableLastModifiedDatetime {
     input:
+      go = CreateAltAlleleTable.done,
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsCreateAltAllele.wdl:32]: bug 2: getting the last modified datetime of the alt allele table should run only after the alt allele table has been created

**Review by RoriCremer — APPROVED** [review]

**Pull request merged**